glacier/src/actions/add-fields.ts

```diff
@@ -0,0 +1,8 @@
+import {ReduxStandardAction} from "./";
+import {Field} from "../model";
+
+export type AddFieldsAction = ReduxStandardAction<"ADD_FIELDS", Field[]>;
```

Comment:
The data source uuid still needs to be incorporated into the payload type and the factory function arguments.

Think of it this way, if I load two DBs, both with  users table and a name field, how would I know which one I'm trying to add or remove? I wouldn't - without specifying which db it was from. That's why the uuid is required.